crates/forge/tests/e2e/common/runner.rs

```diff
@@ -1,19 +1,53 @@
 use assert_fs::fixture::{FileWriteStr, PathChild, PathCopy};
 use assert_fs::TempDir;
 use camino::Utf8PathBuf;
+use fs_extra::dir::{copy, CopyOptions};
 use indoc::formatdoc;
+use once_cell::sync::Lazy;
 use shared::command::CommandExt;
 use shared::test_utils::node_url::node_rpc_url;
 use snapbox::cmd::{cargo_bin, Command as SnapboxCommand};
-use std::path::Path;
+use std::fs::{create_dir_all, remove_dir_all};
+use std::path::{Path, PathBuf};
 use std::process::Command;
 use std::str::FromStr;
 use std::{env, fs};
 use test_utils::tempdir_with_tool_versions;
 use toml_edit::{value, DocumentMut};
 use walkdir::WalkDir;
 
+/// To avoid rebuilding `snforge_std` for each test, we cache it in a directory and copy it to the e2e test temp directory.
```

Comment:
```suggestion
/// To avoid rebuilding `snforge_std` and associated plugin for each test, we cache it in a directory and copy it to the e2e test temp directory.
```